tests/api/testdata/courses_users_enroll_file_val_error.json

```diff
@@ -0,0 +1,44 @@
+{
+    "module": "autograder.api.courses.users.enroll",
+    "read-write": true,
```

Comment:
Note that you can actually mark this test as read only (just leave out this key).
Of course enroll is a r/w operation, but since this test case fails there should be no actual changes.